packit_service/models.py

```diff
@@ -3824,6 +3834,52 @@ def get_by_koji_name(cls, koji_name: str) -> Optional["SidetagModel"]:
             return session.query(SidetagModel).filter_by(koji_name=koji_name).first()
 
 
+class ScanModel(Base):
```

Comment:
```suggestion
class OSHScanModel(Base):
```

so that we are explicit and consistent (e.g. TFT models, Copr,..)